doc/internal/man3/opt.pod

```diff
@@ -0,0 +1,182 @@
+=pod
+
+=head1 NAME
+OPTIONS, OPT_PAIR, OPT_HELP_STR, OPT_MORE_STR, OPT_SECTION_STR,
+OPT_PARAM_STR, opt_progname, opt_appname, opt_getprog, opt_init, opt_format,
+format2str, opt_int, opt_long, opt_imax, opt_umax, opt_ulong, opt_pair,
+opt_string, opt_cipher, opt_md, opt_next, opt_arg, opt_flag, opt_unknown,
+opt_num_rest, opt_rest, opt_help, opt_isdir, opt_progname opt_next
+
+- Option parsing for commands and tests
+
+=head1 SYNOPSIS
+
+ #include "apps/include/opt.h"
```

Comment:
We never include it like that.  Rather, we compile with `-I PATH/TO/apps/include` and then have `#include "opt.h"`